portal/server/api.js

```
@@ -856,11 +856,29 @@ const getPreviousUtilisationReportsByBank = async (token, bankId) => {
   }
 };
 
-const getDueReportDatesByBank = async (token, bankId) => {
+const getMostRecentlyUploadedReportByBank = async (token, bankId) => {
   if (!isValidBankId(bankId)) {
     throw new Error(`Getting due utilisation reports failed for id ${bankId}`);
   }
 
+  const response = await axios({
+    method: 'get',
+    url: `${PORTAL_API_URL}/v1/banks/${bankId}/utilisation-reports/most-recent`,
+    headers: {
+      Authorization: token,
+      'Content-Type': 'application/json',
+    },
+  });
+
+  return response.data;
+};
+
+const getDueReportDatesByBank = async (token, bankId) => {
+  if (!isValidBankId(bankId)) {
+    console.error('Getting due utilisation reports failed for id %s', bankId);
+    return false;
+  }
```

This is a mistake from a rebase - it should throw an error